app/models/facility.rb

```diff
@@ -32,4 +33,11 @@ def image_file_validations
       errors.add(:image_file, I18n.t('activerecord.errors.models.facility.attributes.image_file_dimension'))
     end
   end
+
+  def test
```

Comment:
@leoapost this is not there anymore. Can you still see it? I removed it even before I add you as a reviewer.